### ISSUE TITLE
I'm getting virus warnings on this from virustotal

### ISSUE THREAD

**Issue by montysync** [FoxAhead/Civ2-UI-Additions#31]
![Screenshot 2024-12-27 162637](https://github.com/user-attachments/assets/b26a78b9-e1f6-45f6-81a1-24b593879187)
Just wondering what was going on with this.  Thanks.

**Comment by FoxAhead** [comment]
That is false positive. Civ2UIA is built with Delphi 7 and uses dll-injection and process memory patching, so that altogether triggers AV detection. Most common AV-vendors such as Kaspersky, DrWeb, ESET, Microsoft etc. provide online forms for reporting false-positives and I have already spent a lot of time sending FP-reports to them. But such minor security vendors either don't have such forms, or they don't respond to emails. For example Rising even [terminated English website support](http://www.rising-global.com/) . If you are a user of such vendor, try contacting their support team and report the problem.
This project is open source. You can get the sources and compile it yourself if you have any suspicions.

**Comment by montysync** [comment]
Thanks!
